### ISSUE TITLE
Can't select gist file to upload/download settings

### ISSUE THREAD

**Issue by Nipodemos** [shanalikhan/code-settings-sync#983]
🐛 **Describe the bug**  
I had already used this a few day ago and worked fine. Now I've formatted my PC and trying to use that awnsome plugin to help. But for some random reason it doesn't appear the option to select the gist (nor to skip and create other)
Downloading a public gist worked fine.

🌴 **Visual Studio Code Version :** 1.36.1
🌴 **Code Settings Sync Version :** 3.4.1
🌴 **Standard or Insiders :** standard
🌴 **Portable or Installed :** installed 
🌴 **OSS or Official Build :** I don't know, i just downloaded from VS Code website 
🌴 **Operating System :** Windows 10 May 2019 update x64
🌴 **Occurs On:** First Configuration, when you need to select gist ou skip to create one
🌴 **Proxy Enabled:** no
🌴 **Gist Id:** 42eaf2536d1ad0068b0c5421ebb7a4bf

📰 **To Reproduce**
1. Already have a gist made by settingsSync on other computer
2. install settingssync on your new computer (or on your PC with a fresh new Windows OS)
3. notice that when you click "login with github" it will succeed, but then it will have a blank screen

💪 **Expected behavior**
Select my gist that was already there

📺 **Console Error Log** 
There is no console error

📺  **Additional context**
![image](https://user-images.githubusercontent.com/11494727/61827709-9a819d80-ae3b-11e9-98e4-95e43cbd9584.png)

**Comment by shanalikhan** [comment]
>  There is no console error

@arnohovhannisyan what do you think ? can you look into it.
Can we improve the exception handling for UI.

**Comment by Nipodemos** [comment]
sorry, i must confirm one thing.
I've noticed on other issues that it seems that they look at some developer console of VSCode, i didn't know of this when i posted the issue, i was considering the normal log that is used when we are coding. You want me to test on this developer console to see if something shows up?

**Comment by Nipodemos** [comment]
strange enough, even if i don't have anything to click, since the gist was already created, it is syncing normally when i use the command `shift + alt + u`
honestly have no ideia whats goiing on

```
CODE SETTINGS SYNC UPLOAD SUMMARY
Version: 3.4.1
--------------------
GitHub Token: e2a5425dc069251412b90de2c2070b5597cc4780
GitHub Gist: 1dbc88fb8fe0c3493f042bf25574ce08
GitHub Gist Type: Secret

Restarting Visual Studio Code may be required to apply color and file icon theme.
--------------------
Files Uploaded:
  extensions.json > extensions.json
  keybindings.json > keybindings.json
  settings.json > settings.json
  perl.json > snippets\perl.json

Extensions Ignored:
  No extensions ignored.

Extensions Removed:
  No extensions removed.

Extensions Added:
  better-comments v2.0.5
  bracket-pair-colorizer-2 v0.0.28
  bracket-select v2.0.1
  code-settings-sync v3.4.1
  dart-code v3.2.1
  flutter v3.2.0
  flutter-snippets v2.0.0
  pubspec-assist v0.3.6
  vscode-great-icons v2.1.47
  winteriscoming v1.2.0
--------------------
Done.
```

**Comment by a-wenzel** [comment]
Same here...

**Comment by xyfy** [comment]
Same here...

**Comment by xyfy** [comment]
I just make my gist public ,downloaded then make it private

**Comment by bmaciag** [comment]
same issue here

**Comment by Jahn-Blechinger** [comment]
same issue on windows and ubuntu vsc

**Comment by jjnxpct** [comment]
I also had settings sync installed on one PC and uploaded settings to a gist. And today installed it on another PC. Them I loged in with the GitHub button. After that I saw the 'Select you exisitng Gist'. 

I went to the syn settings and just entered the Gist ID and that worked. It downloaded the settings and everything seems fine. 

>sync: Advanced options > Sync:Open settings -> Enter Gist ID on the left.

After the first sync I did have to enter the Gist ID again somehow. Maybe it wasn't saved or something. But after the second time it stayed and I was able to sync OK. 

Hope this helps?

**Comment by marioarranzr** [comment]
If you set the gistID in configuration page and then restart VSCode, then it works

**Comment by tolgabalci** [comment]
I just installed vscode 1.37.0 on a new machine, and had this same issue.  I just get "Select Your Existing Gist" and nothing is shown below it.  I pressed Alt+Shift+U and a new gist was created.  I had to go Edit Configuration and set the gist id manually to the correct (prior) gist.  Then doing Ctrl+Shift+D worked, but it was a troubling experience.

**Comment by jjnxpct** [comment]
I think you have to add the Gist ID of your existing Gist before any sync actions. But I agree this is not very clear. Setting the Sync up on the first machine is pretty easy. But setting it up on an second machine to sync the first with is less obvious.

**Comment by shanalikhan** [comment]
It is fixed. Will be released in next version.

> Setting the Sync up on the first machine is pretty easy. But setting it up on an second machine to sync the first with is less obvious.

@jjnxpct Do you have any ideas in mind to improve the UX for second machine ? If its regarding the wiki, I would appreciate a PR improving  that (and will release with next version) else you can open new issue so i can look into it in upcoming versions.

**Comment by jjnxpct** [comment]
@shanalikhan I need to set up VSCode on another machine sometime in the next few weeks. I will try to remember to write down what is unclear and could be improved. I think maybe some more text to explain what to do on first setup and what to do if you already have set up on a different machine.

**Comment by shanalikhan** [comment]
It is fixed and released in v3.4.2

**Comment by silviuvulcan** [comment]
Same behaviour for me with fresh install, Sync 3.4.2 and VSCode 1.37.1

After GitHub Login I get a page to select existing Gist with no skip option. 

If I upload then on each upload a new gist is created. I suspect that manually picking up the id of one of them and entering it into settings will work.
